site/content/docs/5.2/utilities/background.md

```diff
@@ -14,11 +14,12 @@ Similar to the contextual text color classes, set the background of an element t
 {{< colors.inline >}}
 {{- range (index $.Site.Data "theme-colors") }}
 <div class="p-3 mb-2 bg-{{ .name }}{{ if .contrast_color }} text-{{ .contrast_color }}{{ else }} text-white{{ end }}">.bg-{{ .name }}</div>
+<div class="p-3 mb-2 bg-{{ .name }}-subtle text-emphasis-{{ .name }}">.bg-{{ .name }}-subtle</div>
 {{- end -}}
 {{< /colors.inline >}}
```

Comment:
Mentioned in https://github.com/twbs/bootstrap/pull/37538, we also have a `.bg-black` (and a `.bg-black.bg-gradient` in Background gradient section).

Moreover we probably want to also add the three following classes: `.bg-body-tertiary`, `.bg-body-secondary` and `.bg-body-emphasis`.